com.unity.render-pipelines.core/Runtime/PostProcessing/Shaders/LensFlareCommon.hlsl

```diff
@@ -125,10 +112,10 @@ float GetOcclusion(float ratio)
         if (all(pos >= 0) && all(pos <= 1))
         {
             float depth0 = GetLinearDepthValue(pos);
-#ifdef UNITY_REVERSED_Z
-            if (_ScreenPosZ < depth0)
+#ifdef SHADER_API_GLES3 // GLES3.hlsl do not define UNITY_REVERSED_Z properly
```

Comment:
This seems a breaking change. GLES2 and GLCore also doesnt have reversed z. 
  What's the issue with REVERSED_Z in GLES3 file? This seems strange as we GLES3 automation on features that depend on reading depth in URP.

Reply:
This PR is merged:
https://github.com/Unity-Technologies/Graphics/pull/5013
So GLES2 is not supported for LensFlareSRP (cf. Discussion with Nigel on Slack).
The define "UNITY_REVERSED_Z" is not defined on GLES3.hlsl (should be 0 or 1).
With this code with URP GLES3:
![rnJ9B5NkcE](https://user-images.githubusercontent.com/55133890/135529528-6a5a0588-2c9d-4e2a-843d-348f65cf25e2.gif)
With URP GLES2 and only this if:
if (COMPARE_DEVICE_DEPTH_CLOSER(depth0, _ScreenPosZ))
![QY77dQMAbR](https://user-images.githubusercontent.com/55133890/135529586-99838fd9-fbbc-4a32-b697-0c0b5e8e1f39.gif)

L.et me know If I missed something.

Reply:
I wonder if we made a mistake here then, the define was removed intentionally so we have both #if and #ifdef usages working in SRP. https://unity.slack.com/archives/C20E70CAC/p1602760055081400

Either way, you should not check for GLES3 as GLCore also needs this, if there's problem with define the GLES3, GLES2 and GLCORE api defines should be fixed instead. 

Let's have a chat over that.

Reply:
[Fixed]